package.json

```diff
@@ -5,11 +5,11 @@
     "dependencies": {
         "@fontsource/noto-sans": "^4.1.0",
         "@material-ui/core": "^4.12.1",
-        "@material-ui/icons": "^4.5.1",
+        "@material-ui/icons": "^4.11.2",
```

Comment:
we can lock the version of this to "@material-ui/core": "~4.11.0"

Reply:
Getting errors `Attempted import error: 'alpha' is not exported from '@material-ui/core/styles'` and many more. Shall I  create a another story and PR for version update?

Reply:
I would check version of material-ui/core that you currently have installed. 
run yarn why material-ui/core

Reply:
yeah, we should stay on `~4.11.0` until we are ready to start doing the upgrade to v5